src/libs/Navigation/getTopmostBottomTabRoute.ts

```diff
@@ -1,7 +1,7 @@
 import type {BottomTabName, NavigationPartialRoute, RootStackParamList, State} from './types';
 
 function getTopmostBottomTabRoute(state: State<RootStackParamList> | undefined): NavigationPartialRoute<BottomTabName> | undefined {
-    const bottomTabNavigatorRoute = state?.routes[0];
+    const bottomTabNavigatorRoute = state?.routes.findLast((route) => route.name === 'BottomTabNavigator');
```

Comment:
Let's use the constant `NAVIGATORS.BOTTOM_TAB_NAVIGATOR`

Reply:
Updated